pcf_test.go

```diff
@@ -38,9 +38,11 @@ func init() {
 	}
 }
 
-/*func init() {
-	factory.InitConfigFactory("amfTest/amfcfg.yaml")
-}*/
+/*
+	func init() {
+		factory.InitConfigFactory("amfTest/amfcfg.yaml")
+	}
+*/
```

Comment:
Why not deleting this function instead of commenting it out?

Reply:
It was already commented out, here I simply ran `go fmt ./...` which formatted the comment correctly. We may want to completely delete it as well as many other unnecessary comments but I see this as a different topic.

Reply:
ok